ci/playbooks/edpm_build_images/run.yml

```diff
@@ -17,11 +17,12 @@
         chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
         cmd: >-
           ansible-playbook ci/playbooks/edpm_build_images/edpm_image_builder.yml
+          -i "{{ ansible_user_dir }}/ci-framework-data/artifacts/zuul_inventory.yml"
+          -e "@{{ ansible_user_dir }}/ci-framework-data/artifacts/zuul_vars.yml"
```

Comment:
same as a previous comment, the param was passed last - might be good to do the same.